### ISSUE TITLE
OFX

### ISSUE THREAD

**Issue by broskees** [dhanasingh/redmine_wktime#144]
I'd love to see the accounting software someday connect to [Open Financial Exchange](http://www.ofx.net/). GnuCash uses it to retrieve transaction history from over [348](http://www.ofxhome.com/index.php/home/directory) banks so far.

I know this is a pipe dream, but is there any possibility of eventual implementation? I've never used OFX personally so I don't know what the scope of this would be. But it would make the accounting side extraordinarily more efficient.

**Comment by dhanasingh** [comment]
yeah, we would like to detailed scope on this to consider for future implementation.  would you like to discuss in detail ? you can send us an email to info@adhisoftware.co.in, then we can setup a call to discuss further

**Comment by broskees** [comment]
To be honest, I really don't know enough about it to be of any help developing a detailed scope for implementation. However, with a bit of searching I found a bit of information that [might help you get started](https://www.bignerdranch.com/blog/online-banking-apis/), and specifically [this library](https://github.com/baconpat/ofx_for_ruby) (though it seems as it may be a bit outdated), which you might find useful. Didn't look into the library enough to see if it's actually a viable option, but it seemed like it could be at a glance.
